components/collapsible-panel/collapsible-panel.js

```diff
@@ -427,23 +420,34 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 	}
 
 	_renderHeader() {
+		const expandCollapseLabel = this.expandCollapseLabel || this.panelTitle;
 		return html`
 			<div class="d2l-collapsible-panel-header" @click="${this._handleHeaderClick}">
 				<div class="d2l-collapsible-panel-before">
 					<slot name="before" @slotchange="${this._handleBeforeSlotChange}"></slot>
 				</div>
 				<div class="d2l-collapsible-panel-header-primary">
-					${this._renderPanelTitle()}
-					<div class="d2l-collapsible-panel-header-actions" @click="${this._handleActionsClick}">
-						<slot name="actions"></slot>
-					</div>
-					<div class="d2l-collapsible-panel-opener">
+
+					<div
```

Comment:
Looks like the keyboard a11y is broken. `[Enter]` and `[Space]` should toggle the expanded state of the panel. 

One of the challenges with rolling our our "button" (via `role="button"`) is that we need to program all the behaviours ourselves. In this case you'd need to wire up for a key event, check for those keys, and toggle if appropriate. For that reason, we usually prefer to use the best semantic element that the browser natively provides.

Is there a reason we can't use `<button>` here? The flip-side of the coin is that if we use `<button>`, we need to override styles (at minimum `border: none; background-color: transparent;`) since we'd only be using the button element for the semantics and behaviour that it provides.

Also note: if handling `[Space]`: we would probably want to prevent the default scroll behaviour...
```javascript
_handleKeyDown(e) {
	if (e.keyCode === 32) e.preventDefault(); // don't scroll the page when space is pressed
}
_handleKeyUp(e) {
	if (e.keyCode !== 32 && e.keyCode !== 13) return;
	... // toggle expanded state...
}
```